app/services/sqs/publish_laa_reference.rb

```diff
@@ -2,6 +2,15 @@
 
 module Sqs
   class PublishLaaReference < ApplicationService
+    TEMPORARY_CJS_AREA_CODE = 16
+    TEMPORARY_CJS_USER = 'cpUser'
+    TEMPORARY_CJS_LOCATION = 'B16BG'
```

Comment:
I think maybe `TEMPORARY_USER` might be a better name here. I _think_ that the user details we're interested here are internal LAA users, so wouldn't really fall under CJS. No big deal but might just avoid confusion.

Reply:
Good point, It is definitely clearer.